ui/src/admin/components/UserRow.js

```diff
@@ -69,6 +75,9 @@ const UserRow = ({
             /> : null
         }
       </td>
+      <td className="text-right" style={{width: "300px"}}>
+        <ChangePassRow onEdit={onEdit} onApply={handleUpdatePassword} user={user} />
```

Comment:
`ChangePassRow` is a bit unclear on what it is -- thoughts on a better name for this component, possibly? I faced this same question when moving from `DeleteRow` to `DeleteConfirmTableCells` + `DeleteConfirmButtons`, but yours also has an input field...